Add tmux commands to use in terminal

ma-anwar opened the pull request:
Not sure if prefixing every command with mux is a good idea?

fidgetingbits commented:
I think using `mux` is good

knausj85 commented:
Pardon my ignorance, is this a standard feature of linux terminals, or would a tag make sense here?

ma-anwar commented:
No, this is not a standard feature. Did I use the tag incorrectly? Sorry I still get mixed up, should it be,
`tag(): terminal`?

knausj85 commented:
No, you used tag correctly - I meant: does a custom, separate tag from terminal (tmux or something), make sense here that could be asserted as-needed by terminal programs? Particularly since some apps might opt into some ‘standard’ terminal stuff (e.g. vscode).

I really know nothing about this one, just asking dumb questions. 🤔

ma-anwar commented:
Yes definitely a custom tag makes sense in this case. I was not aware of vscode opting into terminal commands as well but that makes sense.

knausj85 commented:
Thanks. This tag needs to be declared... somewhere, no? Maybe you missed a file? :)

e.g., https://github.com/knausj85/knausj_talon/blob/master/code/gdb.py

It probably makes sense to have a /code/tags.py for something like this, where we can just toss tags that need no other code support

ma-anwar commented:
Thank you for teaching me the perils of testing on my own fork as opposed to the patch branch! The tags file is a useful idea. I have added a tags file that declares tags with generic descriptions. I think I managed to collect the custom tags defined so far. 

I also added a line to declare modes, but I'm not sure if that should be in the scope of this file. Also not sure if mode description should be generic as well,

knausj85 commented on apps/linux/tmux.talon:
```diff
@@ -0,0 +1,55 @@
+    insert('tmux new ')
+mux sessions:
+    key(ctrl-b)
+    key(s)
```
This is just an FYI as I'll be merging this as-is, but you should be able to combine some of these into one command, unless you need arbitrary delay between
e.g.,
`key(ctrl-b s)`
`key("ctrl-b {number}")`
